src/handlers/swap_handlers.ts

```diff
@@ -37,10 +37,24 @@ export class SwapHandlers {
             excludedSources,
             affiliateAddress,
         } = parseGetSwapQuoteRequestParams(req);
+
         const isETHSell = isETHSymbol(sellToken);
+        // if sellToken is not WETH and buyToken is ETH, throw
+        if (!isWETHSymbol(sellToken) && isETHSymbol(buyToken)) {
```

Comment:
I think `sellToken`  can be an address here, not always a symbol. We should handle `isWETHSymbol(WETH_ADDRESS)` as `sellToken=0xc02&buyToken=ETH` should work as expected

```suggestion
        if (!isWETHSymbolOrAddress(sellToken) && isETHSymbol(buyToken)) {
```